src/hooks/useFilterCriteria.ts

```diff
@@ -38,6 +38,7 @@ export const useFilterCriteria = (): useFilterCriteriaReturns => {
 
   useEffect(() => {
     const allProposals = daoStore.getAllProposals();
+
     // (QuitedEndingPeriod || Queded) && positiveVotes >= 10% (Ordered from time to finish, from lower to higher)

```

Comment:
@hhamud if you end up removing that criteria in the filter, change this comment. 
cheers!